### PR TITLE
Added MemeUrl

### PR DESCRIPTION
**Pull request by mohammedjameel24**
### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
image

### CODE REVIEW

**Comment by mohammedjameel24** [comment]
merge my pull request @ridsuteri

**Comment by mohammedjameel24** [comment]
merge

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@mohammedjameel24 this project is not working on my side...
please check once again and do the changes ,if required

**Comment by mohammedjameel24** [comment]
> @mohammedjameel24 this project is not working on my side...
> please check once again and do the changes ,if required

Is it not even displaying of not creating posts?

**Comment by mohammedjameel24** [comment]
commited into different pull #700

File: script.js [hunk 1]
```diff
@@ -205,6 +205,20 @@ let projectData = [
...
+    projectName: "MemeUrl",
+    projectImage: "assets/GIFs/memeUrl.png",
+    projectUrl: "public/memeUrl.html",
+  },
+  {
+    projectName: "RGB Color Guessing Game",
+    projectImage: "assets/GIFs/RGB_Color.gif",
+    projectUrl: "public/RGB_Color_Guessing_Game.html",
+  },
     projectName: 'Animated Minion Eyes',
```
Comment (ridsuteri) [inline script.js:222]: { is misssing after line 221

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@mohammedjameel24 there is some issue with script file ,please resolve it and check locally before putting a PR ,rest your project is working fine :)

**Comment by mohammedjameel24** [comment]
now?

**Comment by mohammedjameel24** [comment]
Please merge my pull request

**Comment by mohammedjameel24** [comment]
Please merge my pull request@ridsuteri

**Review by ridsuteri — CHANGES_REQUESTED** [review]
nopes @mohammedjameel24 still not working properly...
Your script file is creating problems...
Please check locally on your system before opening a PR

**Comment by mohammedjameel24** [comment]
Memeurl.js ? Or script.js?

**Comment by ridsuteri** [comment]
@mohammedjameel24 script.js

**Comment by mohammedjameel24** [comment]
please tell me now?

**Review by ridsuteri — APPROVED** [review]
done @mohammedjameel24 ,thanks for contributing :)

**Pull request merged**